src/main/java/Lotto/domain/Lottos.java

```diff
@@ -9,14 +9,38 @@ public class Lottos {
     private static final int LOTTO_MAX_NUMBER = 45;
     private static final int LOTTO_NUMBER_SIZE = 6;
 
-    public Lottos(int lottoCount) {
+    private int passiveLottoCnt;
+    private int automaticLottoCnt;
+
+    public Lottos(Lotto [] lottos) {
+        for (Lotto lotto : lottos) {
+            list.add(lotto);
+        }
+    }
+
+    public Lottos(int automaticLottosCnt, List<Lotto> passiveLottos) {
+        initLottoCount(automaticLottosCnt, passiveLottos.size());
+
         List<Integer> tmpNumbers = createAllNumbers();
 
-        for (int i = 0; i < lottoCount; ++i) {
+        addPassiveLottos(passiveLottos);
+
+        for (int i = 0; i < automaticLottoCnt; ++i) {
             pickLottoNumbers(tmpNumbers);
         }
     }
 
+    private void initLottoCount(int automaticLottosCnt, int passiveLottoCnt) {
+        this.automaticLottoCnt = automaticLottosCnt;
+        this.passiveLottoCnt = passiveLottoCnt;
+    }
+
+    private void addPassiveLottos(List<Lotto> passiveLottos) {
+        for (int i = 0; i < passiveLottos.size(); ++i) {
+            list.add(passiveLottos.get(i));
+        }
```

Comment:
```suggestion
        list.addAll(passiveLottos)
```
addAll() 메소드를 활용해도 좋을 것 같아요 :)

Reply:
이런 메소드가 있군요